gef-ui/templates/partials/upload-supporting-documents.njk

```diff
@@ -108,5 +109,23 @@
 {% endblock %}
 
 {% block scripts %}
-  <script src="/assets/js/fileUpload.js"></script>
+  <script>
+    if (typeof MOJFrontend.MultiFileUpload !== 'undefined') {
+      const onFileDelete = (mojFileUpload, response) => {
+        if (response.error) {
+          const errorRow = `<div class="govuk-summary-list__row moj-multi-file-upload__row moj-multi-file-upload__error-row"><div class="govuk-summary-list__value moj-multi-file-upload__message">${mojFileUpload.getErrorHtml(response.error)}</div></div>`;
```

Comment:
Is there no other cleaner way of presenting an error message back to the component?

Reply:
I couldn't find a better way of doing it.

The delete method in `mojFileUpload` didn't seem to handle showing error messages, so the best way I could find to hook in was in this `fileDeleteHook`.

And then `mojFileUpload.getErrorHtml` was the biggest bit of the other error messages HTML I could get directly from `mojFileUpload`. So I've added the  other divs and classes are so it matches the other rows.